 01-Makefile.diff by Neal H Walfield <neal@cs.uml.edu> and
 Gergely Nagy <algernon@debian.org>

 Add the necessary includes and other things to the libedit
 Makefile, so it will compile on GNU systems, with our supplied
 glue code

--- a/libedit/Makefile
+++ b/libedit/Makefile
@@ -1,6 +1,13 @@
 #	$NetBSD: Makefile,v 1.36 2007/05/28 12:06:18 tls Exp $
 #	@(#)Makefile	8.1 (Berkeley) 6/4/93
 
+OS!=	sh -c "case `uname -s` in GNU|GNU/*) echo GNU ;; *) uname -s ;; esac"
+
+.if ${OS} == GNU || ${OS} == Linux
+CFLAGS += -include bsd/bsd.h
+MKLINT=no
+.endif
+
 USE_SHLIBDIR=	yes
 
 WARNS=	3
